spec/lib/api/v3/projects/schemas/project_schema_representer_spec.rb

```diff
@@ -182,7 +182,7 @@
       let(:path) { 'status' }
 
       it_behaves_like 'has basic schema properties' do
-        let(:type) { 'ProjectStatus' }
```

Comment:
We currently rely on that type in the frontend: https://github.com/opf/openproject/blob/dev/frontend/src/app/modules/fields/display/display-field.initializer.ts#L82